docs/Remix.md

```diff
@@ -295,23 +336,28 @@ The stylesheet link is necessary to reset the default styles of the admin app. C
 body { margin: 0; }
 ```
 
-Remix and react-admin both use [react-router](https://reactrouter.com/) for routing. React-admin detects when it is included inside an existing React Router context and reuses it. This is problematic because Remix uses file-based routing. So when react-admin changes the route to `/posts` for instance, Remix will look for a corresponding `app/routes/posts.tsx` file. As it doesn't exist, Remix will render a 404.
+**Tip**: Remix doesn't let splat routes catch requests to the index page ('/'), so you need to have both the `app/routes/index.tsx` and `app/routes/$.tsx` files to correctly render the admin app.
```

Comment:
This is duplicate to what's said in the beginning of the paragraph. Besides, it's `app/routes/_index.tsx` and not `app/routes/index.tsx`.